php/class-pwawp-app-manifest.php

```diff
@@ -0,0 +1,171 @@
+<?php
+/**
+ * PWAWP_APP_Manifest class.
+ *
+ * @package PWA
+ */
+
+/**
+ * PWAWP_APP_Manifest class.
+ *
+ * Mainly copied from Jetpack_PWA_Manifest and Jetpack_PWA_Helpers.
+ */
+class PWAWP_APP_Manifest {
```

Comment:
Let's name things as if they are going to be merged into core, so this would be `WP_Web_App_Manifest`

Reply:
Sure, [6df56](https://github.com/xwp/pwa-wp/pull/13/commits/6df5656d56a0f2c9e4893b63d326be6f1369ee15) renames the classes.